### DISCUSSION TITLE
I'm lost with trying to install SavvyCAN on my Kali Linux laptop and really would pay someone to do after wasting this much time.

### DISCUSSION THREAD

**Discussion by Tugboat-Fondren** [collin80/SavvyCAN#792] [General]
I'm not a noob so much as I have lost my mind. Look, I've been down so many rabbit holes because I've followed so much bad advice and because I'm constantly exhausted from working the 3rd shift as a foreman. I couldn't tell you where I am on trying to get SavvyCAN installed on a laptop that I put Kali Linux on two or three weeks ago. It's not that having SavvyCAN and Kali Linux is some life long ambition of mine, I just need to see what's going on with this Android head unit in my truck and why my truck's can bus system is unable to identify the head unit, then maybe I'll do something about it.
I've tried to follow step by step guides, they all seem to get to some arbitrary point where my setup doesn't want to do whatever some person on YouTube or in a forum has done so very easily. It's been over a month that I've attempted to do the simplest link into the vehicle's can bus network to determine what is not happening, but I've gone backwards by destroying my good and primary laptop by following the advice of an idiot I work with. 
So, I've determined that I've found every guide on SavvyCAN and attempted to follow them to no success. What else can I do? 
I do have a separate github account, but like I said in the beginning, I've lost my mind and its embarrassing to be where I am asking how to even get this software on my laptop. When an error returns, I stare at it for hours with a complete blank expression because I'm exhausted and no longer have the will for going down any more damn rabbit holes chasing what is wrong with some other package or syntax or anything.

**Comment by tayloraleach** [comment]
Do you want to just run the program? Can you not just download the standalone appimage from the releases page? https://github.com/collin80/SavvyCAN/releases

As of right now, this is the latest release:
https://github.com/collin80/SavvyCAN/releases/download/V213/SavvyCAN-044fea3-x86_64.AppImage

You may need to google around on how to launch an appimage on linux but that should not be too difficult
